### PR TITLE
Remove unused gems and update others

### PR DESCRIPTION
**Pull request by vitogit**
This project was forked from https://github.com/rootstrap/rails_api_base/ but there are a lot of gems that are not used. 

I'm removing those and also updating some other (vulnerabilities).

If vulnerabilities still persist I will create a new PR to upgrade rails

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,35 +1,24 @@
...
-gem 'devise', '~> 4.3.0'
-gem 'devise_token_auth', git: 'https://github.com/lynndylanhurley/devise_token_auth'
-gem 'draper', '~> 3.0.0'
-gem 'fog-aws', '~> 2.0.0'
-gem 'haml-rails', '~> 1.0.0'
-gem 'jbuilder', '~> 2.7.0'
-gem 'koala', '~> 1.10.1'
-gem 'oj', '~> 2.17.5'
+gem 'devise', '~> 4.7.1'
+gem 'devise_token_auth'
```
Comment (fedeagripa) [inline Gemfile]: can we add version here too?

**Review by glm4 — APPROVED** [review]

File: app/assets/config/manifest.js [hunk 2]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (fedeagripa) [inline app/assets/config/manifest.js]: maybe i'm just out of scope here, but why this?

**Review by jpascual1994 — APPROVED** [review]

File: Gemfile [hunk 3]
```diff
@@ -1,35 +1,24 @@
 source 'https://rubygems.org'
-ruby '2.3.1'
+ruby '2.6.3'
```
Comment (santib) [inline Gemfile]: can we use `~>` ? https://andycroll.com/ruby/use-loose-ruby-versioning-in-your-gemfile/

File: app/assets/config/manifest.js (hunk 2 above)
Comment (santib) [reply app/assets/config/manifest.js]: the Rails default template is this
```
//= link_tree ../images
//= link_directory ../stylesheets .css
```

Comment (vitogit) [reply app/assets/config/manifest.js]: Yes, but we are not using assets, I would like to remove everything related to assets, but if I remove `require "sprockets/railtie"` then I receive this error https://github.com/fgrehm/letter_opener_web/issues/46
Also  Sprockets 4. expect that file https://github.com/rails/sprockets/blob/master/UPGRADING.md#manifestjs

File: Gemfile (hunk 1 above)
Comment (vitogit) [reply Gemfile]: yes

File: Gemfile (hunk 3 above)
Comment (vitogit) [reply Gemfile]: yes

File: app/assets/config/manifest.js (hunk 2 above)
Comment (santib) [reply app/assets/config/manifest.js]: aren't we using assets in active admin?

Comment (vitogit) [reply app/assets/config/manifest.js]: I totally forgot that we were using active admin. Good catch

**Review by santib — APPROVED** [review]
👏

**Review by fedeagripa — APPROVED** [review]

**Pull request merged**